graph/types.go

```diff
@@ -13,6 +13,7 @@ const (
 	GraphTypeVersionedApp string = "versionedApp"
 	GraphTypeWorkload     string = "workload"
 	NodeTypeApp           string = "app"
+	NodeTypeOperation     string = "operation" // The special request_operation aggregate node
 	NodeTypeService       string = "service"
```

Comment:
This is unrelated to this PR (which should just be about removing the component namespace stuff) so this should be reverted. This "operation" thing is for the upcoming work on the experimental metric classification stuff.

Reply:
good catch, looks like I had PR spillage... Fixed.